### ISSUE TITLE
CLI is not working

### ISSUE THREAD

**Issue by hadooping** [ponzu-cms/ponzu#341]
➜  ~ ponzu
zsh: command not found: ponzu

**Comment by hadooping** [comment]
After installing Ponzu, CLI is not working

➜  ~ go get -u github.com/ponzu-cms/ponzu/...
➜  ~
➜  ~
➜  ~ ponzu
zsh: command not found: ponzu

**Comment by khozaei** [comment]
Please run the command below in terminal and then use `ponzu`
<pre>
export PATH=`go env GOPATH`/bin:$PATH
</pre>

**Comment by hadooping** [comment]
Thanks @khozaei but this is not working.

**Comment by make0x20** [comment]
Hi,

Can confirm that the command is not working for me either after running go get..

Any ideas on any other steps to try?

Thanks

**Comment by dmitridb** [comment]
> Hi,
> 
> Can confirm that the command is not working for me either after running go get..
> 
> Any ideas on any other steps to try?
> 
> Thanks

`$ echo $SHELL`
`$ echo $PATH`
`$ echo $GOPATH`

what do these say for you?

**Comment by make0x20** [comment]
$SHELL
`/usr/bin/zsh`

$PATH
`/home/tim/.nvm/versions/node/v12.13.1/bin:/home/tim/go/bin:/usr/local/go/bin/:/home/tim/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin:/home/tim/Development/Bash`

$GOPATH
`/home/tim/go`

**Comment by dmitridb** [comment]
Try running it from bash instead (making sure necesary path is the same)

**Comment by make0x20** [comment]
Running it from bash did not work for me either but I did refer to https://github.com/ponzu-cms/ponzu/issues/342 issue. 

I checkedout to `v11.x` branch on **github.com/blevesearch/zap** locally and ran `go get -u github.com/ponzu-cms/ponzu/...` again. That seemed to make it work.

**Comment by mvritlimited** [comment]
Same case, not sure if this is supported anymore.

**Comment by hadooping** [comment]
Same case, not sure if this is supported, really frustrating.

**Comment by dmitridb** [comment]
Yeah well all this $GOPATH stuff was deprecated along with the changes to the module system, so this entire project needs a fix to that end. I'm spending some of my spare time trying to figure out how to best accomplish this. If you get a better idea of how to do this before I do lemme know because all I got was a big shrug from @nilslice

**Comment by dmitridb** [comment]
It's been hooped since august 2019 with go 1.13

**Comment by dmitridb** [comment]
If you hope to understand further this goes into pretty complete details by the way https://github.com/golang/go/wiki/Modules

**Comment by sysfu** [comment]
@dmitridb I'm stuck getting ponzu to install using the fish shell. If you want any help testing the codebase updates lmk. Also curious if you've seen any interest or signs of life from @apilayer. I sent an email to support@apilayer.com and received and auto-responder, but no signs up human life yet.

**Comment by pavelkrolevets** [comment]
> Running it from bash did not work for me either but I did refer to #342 issue.
> 
> I checkedout to `v11.x` branch on **github.com/blevesearch/zap** locally and ran `go get -u github.com/ponzu-cms/ponzu/...` again. That seemed to make it work.

This worked for me. Also manually added to **github.com/blevesearch/zap** folders **v12**, **v13**, **v14** and unzip there from **zap** releases axives.
